packages/kit/src/core/sync/write_env.js

```diff
@@ -29,6 +30,7 @@ export function write_env(config, mode) {
 	write_if_changed(
 		path.join(config.outDir, 'ambient.d.ts'),
 		autogen_comment +
+			types_reference +
```

Comment:
Perhaps we should rename `write_env`, since it's not only about writing env types now. Maybe `write_ambient`? Other suggestions?

Reply:
I like `write_ambient`!

Reply:
Renamed function and module to `write_ambient`. Also, changed names of the helpers related to env.